docs/releases/3.2.txt

```diff
@@ -681,6 +681,13 @@ Miscellaneous
   ``UserChangeForm.clean_password()`` is no longer required to return the
   initial value.
 
+* The ``.get_many()``, ``.get_or_set()``, ``.has_key()``, ``.incr()``,
+  ``.decr()``, ``.incr_version()``, and ``.decr_version()`` cache operations
+  now correctly handle ``None`` stored in the cache in the same way as any
+  other value instead of behaving as though the key didn't exist. Due to
+  deficiencies in the underlying client library the previous behavior will
```

Comment:
Should we be quite so negative here? We could say say something along the lines of 'due to the implementation in the underlying...'

Reply:
Updated.